_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -26,101 +26,34 @@ To enable searchable snapshots, reference the [instructions](https://opensearch.
 
 Note that searchable snapshots cannot be deleted until the `searchable` flag on these snapshots is removed. Follow the snapshot [configuration steps]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/) in order to remove the flag.
 
 ## Benchmarks
 
-For the following benchmarks, the OpenSearch benchmark workload was configured with 0 warmup iterations so that the cache was not populated when the measurements started. The test aimed to achieve 1.5 operations per second. `search_clients` was configured to 16 to match the number of CPU cores on the hosts. For searchable snapshots, the test was performed immediately after creating the searchable snapshot index. For UltraWarm, the test was performed immediately after migrating the index to the warm tier so that the cache was minimally warmed in both test cases.
-
-<table><colgroup><col /><col /><col /><col /><col /></colgroup>
-<tbody>
-<tr>
-<td><strong>Normal Index</strong></td>
-<td><strong>Searchable Snapshot</strong></td>
-<td><strong>Searchable Snapshot (single segment)</strong></td>
-<td><strong>UltraWarm</strong></td>
-</tr>
-<tr>
-<td><strong>Throughput (search ops/s)</strong></td>
-</tr>
-<tr>
-<td>min</td>
-<td>0.45</td>
-<td>0.05</td>
-<td>0.05</td>
-<td>0.05</td>
-</tr>
-<tr>
-<td>mean</td>
-<td>1.48</td>
-<td>1.38</td>
-<td>1.40</td>
-<td>1.38</td>
-</tr>
-<tr>
-<td>median</td>
-<td>1.50</td>
-<td>1.45</td>
-<td>1.45</td>
-<td>1.45</td>
-</tr>
-<tr>
-<td>max</td>
-<td>1.50</td>
-<td>1.47</td>
-<td>1.48</td>
-<td>1.47</td>
-</tr>
-<tr>
-<td><strong>Latency (ms)</strong></td>
-</tr>
-<tr>
-<td>p50</td>
-<td>1,065.84</td>
-<td>1,111.07</td>
-<td>1,163.01</td>
-<td>1,115.59</td>
-</tr>
-<tr>
-<td>p90</td>
-<td>1,099.68</td>
-<td>1,162.98</td>
-<td>1,304.18</td>
-<td>1,228.05</td>
-</tr>
-<tr>
-<td>p99</td>
-<td>1,177.59</td>
-<td>11,287.20</td>
-<td>10,165.70</td>
-<td>10,827.60</td>
-</tr>
-<tr>
-<td>p99.9</td>
-<td>2,212.63</td>
-<td>20,595.50</td>
-<td>19,323.90</td>
-<td>20,186.10</td>
-</tr>
-<tr>
-<td>p100</td>
-<td>2,212.72</td>
-<td>20,596.00</td>
-<td>19,324.00</td>
-<td>20,186.40</td>
-</tr>
-</tbody>
-</table>
-
-In OpenSearch 2.7, you can now search snapshots locally on your OpenSearch cluster with performance similar to UltraWarm storage. With searchable snapshots, you can search through less frequently accessed indexes saved as snapshots without having to restore them to the cluster beforehand. 
+For the following benchmarks, the OpenSearch benchmark workload was configured with 0 warmup iterations so that the cache was not populated when the measurements started. The test aimed to achieve 1.5 operations per second. `search_clients` was configured to 16 to match the number of CPU cores on the hosts. For searchable snapshots, the test was performed immediately after creating the searchable snapshot index. 
+
+| | Normal Index | Searchable Snapshot | Searchable Snapshot (single segment) | 
+| Throughput (search ops/s) | | 
```

Comment:
The table is hard to read. Can we add grid lines and format more clearly the headers of each section "Throughput (search ops/s)" and "Latency (ms)"

Reply:
Done